### ISSUE TITLE
Jobs error in console

### ISSUE THREAD

**Issue by mooondark** [quisquous/cactbot#3819]
Since few days playing WAR, I get the following error (a lot) :
```
[12/12/2021 19:07:59] Info: Job: BrowserConsole: Error: This code shouldn't be reached (Source: file:///C:/Applications/ACT%20Plugins/cactbot/ui/jobs/jobs.bundle.js, Line: 2083)
[12/12/2021 19:08:02] Info: Job: BrowserConsole: Error: This code shouldn't be reached (Source: file:///C:/Applications/ACT%20Plugins/cactbot/ui/jobs/jobs.bundle.js, Line: 2083)
```
I attach the log if needed.
[Network_26208_20211212.zip](https://github.com/quisquous/cactbot/files/7699305/Network_26208_20211212.zip)

**Comment by quisquous** [comment]
This should be fixed by #3800.

**Comment by quisquous** [comment]
If this happens again, please comment back here, but it should be fixed.
